src/database/simple.jl

```diff
@@ -338,64 +338,70 @@ match(SimpleRename, CSTParser.IDENTIFIER) do x
-    @add_rename method_exist   hasmethod        v"0.7.0-DEV.3455"
-    @add_rename object_id      objectid         v"0.7.0-DEV.3455"
-    @add_rename DevNull        devnull          v"0.7.0-DEV.4068"
-    @add_rename STDIN          stdin            v"0.7.0-DEV.4068"
-    @add_rename STDOUT         stdout           v"0.7.0-DEV.4068"
-    @add_rename STDERR         stderr           v"0.7.0-DEV.4068"
+    @add_rename select         partialsort      v"0.7.0-DEV.1535" true
+    @add_rename select!        partialsort!     v"0.7.0-DEV.1535" true
+    @add_rename selectperm     partialsortperm  v"0.7.0-DEV.1535" true
+    @add_rename selectperm!    partialsortperm! v"0.7.0-DEV.1535" true
+    @add_rename Range          AbstractRange    v"0.7.0-DEV.1721" true
+    @add_rename isleaftype     isconcretetype   v"0.7.0-DEV.1775" true
+    @add_rename isnumber       isnumeric        v"0.7.0-DEV.1775" true
+    @add_rename Associative    AbstractDict     v"0.7.0-DEV.2951" true
+    @add_rename find           findall          v"0.7.0-DEV.3415" true
+    @add_rename module_parent   parentmodule    v"0.7.0-DEV.3460" true
+    @add_rename datatype_module parentmodule    v"0.7.0-DEV.3460" true
+    @add_rename function_module parentmodule    v"0.7.0-DEV.3460" true
+    @add_rename indmin         argmin           v"0.7.0-DEV.3516" true
+    @add_rename indmax         argmax           v"0.7.0-DEV.3516" true
+    @add_rename module_name    nameof           v"0.7.0-DEV.3539" true
+    @add_rename function_name  nameof           v"0.7.0-DEV.3539" true
+    @add_rename datatype_name  nameof           v"0.7.0-DEV.3539" true
+    @add_rename method_exist   hasmethod        v"0.7.0-DEV.3455" true
+    @add_rename object_id      objectid         v"0.7.0-DEV.3455" true
+    @add_rename DevNull        devnull          v"0.7.0-DEV.4068" true
+    @add_rename STDIN          stdin            v"0.7.0-DEV.4068" true
+    @add_rename STDOUT         stdout           v"0.7.0-DEV.4068" true
+    @add_rename STDERR         stderr           v"0.7.0-DEV.4068" true
     @add_rename reprmime       repr             v"0.7.0-DEV.4010"
     @add_rename parse          Meta.parse       v"0.7.0-DEV.2437"
-    @add_rename isabstract     isabstracttype   v"0.7.0-DEV.1775"
-    @add_rename iteratorsize   IteratorSize     v"0.7.0-DEV.3309"
-    @add_rename iteratoreltype IteratorEltype   v"0.7.0-DEV.3309"
-    @add_rename nb_available   bytesavailable   v"0.7.0-DEV.3477"
+    @add_rename isabstract     isabstracttype   v"0.7.0-DEV.1775" true
+    @add_rename iteratorsize   IteratorSize     v"0.7.0-DEV.3309" true
+    @add_rename iteratoreltype IteratorEltype   v"0.7.0-DEV.3309" true
+    @add_rename nb_available   bytesavailable   v"0.7.0-DEV.3477" true
     @add_rename broadcast_indices broadcast_axes  v"0.7.0-DEV.4936"
 ]
 
+for (ver, repl, hascompat) in renames
+    if hascompat
+        match(SimpleRename, "Compat.$(repl.second)", repl.second)
```

Comment:
These are a bit expensive, I'd prefer this being written out manually in the `match(SimpleRename, CSTParser.IDENTIFIER) do x`. Let me know if you know how to do that or want me to do it.

Reply:
I'll have a look but I'm busy in the upcoming days so feel free to do it.

Reply:
I tried this, but the problem is that some of the replacements are not merely simple identifiers which makes this harder. Since this works, I'll got ahead and merge this. Thanks!